Makefile

```diff
@@ -65,9 +65,11 @@ build_python:: schema
 		cd ./bin && python3 setup.py build sdist
 
 build_go:: VERSION := $(shell pulumictl get version --language generic)
+build_go:: AWS_VERSION := $(shell node -e 'console.log(require("./awsx/package.json").dependencies["@pulumi/aws"])')
```

Comment:
Probably could've used jq, but couldn't remember if that's available on CI already 🤷

Reply:
Oh nice!!